requirements.txt

```
@@ -1,3 +1,4 @@
 future>=0.15.2
 jinja2>=2.8
-urllib3 >= 1.15
+urllib3>=1.15
+isi_sdk_7_2>=0.1.2
```

0.2.x contains backwards-incompatible changes (see #19).
```suggestion
isi_sdk_7_2~=0.1.2
```
[The compatible release operator](https://www.python.org/dev/peps/pep-0440/#compatible-release) should cause the latest _compatible_ `isi_sdk_7_2` to be installed ([currently, 0.1.11](https://pypi.org/project/isi-sdk-7-2/#history)).